src/main/java/org/jabref/gui/fieldeditors/contextmenu/EditorMenus.java

```diff
@@ -84,4 +85,24 @@
             return menuItems;
         };
     }
+
+    /**
+     * The default context menu with a specific menu item to cleanup  URL.
+     *
+     * @param textArea text-area that this menu will be connected to
+     * @return menu containing items of the default menu and an item to cleanup a URL
+     */
+    public static Supplier<List<MenuItem>> getCleanupURLMenu(TextArea textArea) {
+        return () -> {
+            CustomMenuItem cleanupURL = new CustomMenuItem(new Label(Localization.lang("Cleanup URL Link")));
+            //cleanupURL.setDisable(true);
```

Comment:
remove comment or better bind the disable property to `textArea.textProperty().isEmpty()`